changelog.md

```diff
@@ -9,7 +9,7 @@
   - iOS touchable highlight underlay tokens
   - iOS shadow tokens
   - iOS & Android radii tokens
-
+  - Update Android text tokens
```

Comment:
Massive nitpick: 

Can we have a newline between components in the changelog? 😇